.github/workflows/format-suggester.yml

```diff
@@ -0,0 +1,18 @@
+name: Format suggester
+on: pull_request
+jobs:
+  suggest-gofmt:
+    name: Suggest gofmt changes
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+
+      - name: Run gofmt
+        run: gofmt -s -w .
+
+      - name: Suggest gofmt changes # in the form of a PR review
+        uses: reviewdog/action-suggester@v1
+        with:
+          tool_name: gofmt
+          filter_mode: nofilter
```

Comment:
do we need this? seems easier to just run `go fmt .` and make a single commit

Reply:
Well, that is a good point. I suppose I was thinking of the "Why?" instead of the "Why not?" 

Doing one format commit instead of accepting a bunch of different suggestions should be less tedious and should avoid relying on the bot as a crutch.

It's more consistent and less bloat to just always look at the failed format checker workflow and know that you have to format.